### DISCUSSION TITLE
Qingping CGDK2 display strange measures!

### DISCUSSION THREAD

**Discussion by raykung12** [pvvx/ZigbeeTLc#42] [Q&A] [answered]
Need help with Qingping CGDK2.

I have two CGDK2. OTA first one with ZCGDK2-0118 firmware, pair into HA without any issue! But OTA 2nd one with 0118 ran into strange issue : display stuck at -3.9°C and 10%RH (see photo),  pair into HA ok but with same fixed readings. I tried to OTA back to BLE and repeat the procedures many times with the same result. Appreciate for any suggestion and guidance, thanks for the great work!

![IMG_20240103_015644](https://github.com/pvvx/ZigbeeTLc/assets/8249358/8bc39dc0-a221-47c0-a02a-91435e82bdf4)

**Comment by pvvx** [comment, marked as answer]
This is a variant of CGDK2 with a different sensor.
https://github.com/pvvx/ZigbeeTLc/issues/27
